src/lumigo_tracer/sync_http/sync_hook.py

```diff
@@ -313,4 +328,6 @@ def wrap_http_calls():
                 wrap_function_wrapper(
                     "urllib3.response", "HTTPResponse.read_chunked", _read_stream_wrapper
                 )
+            if importlib.util.find_spec("requests"):
+                wrap_function_wrapper("requests.api", "request", _requests_wrapper)
```

Comment:
Won't there a double wrapping for the request?

Reply:
there is!
Note that `_requests_wrapper` (the "higher" hook) doesn't create span by itself, but only append data to the span that was created with `http.client` (the lower hook).